zc_install/sql/updates/mysql_upgrade_zencart_200.sql

```diff
@@ -178,6 +178,9 @@ UPDATE configuration SET configuration_description = 'What is the weight of typi
 # Add template_settings field
 ALTER TABLE template_select ADD template_settings LONGTEXT DEFAULT NULL;
 
+# Change minimum dob field length for new date VALIDATION
+UPDATE configuration SET configuration_value = 8 WHERE configuration_key = 'ENTRY_DOB_MIN_LENGTH';
```

Comment:
Question: Should we change it only if it's currently at the previous default of `10`?
```suggestion
UPDATE configuration SET configuration_value = 8 WHERE configuration_key = 'ENTRY_DOB_MIN_LENGTH' AND configuration_value=10;
```